### PR TITLE
Minor tweaks in Facility

### PR DESCRIPTION
**Pull request by ProCode2**
## Proposed Changes

- Minor changes in facilities
    - Improved members page UI (by @Apurva-Nagar ).
    - updated `patients` link in facilities tab to point to `/patients`.
    - changed and SVG to a font-awesome icon.

## Merge Checklist

- [x] Check if route, query, or mutation authorization looks correct.

### CODE REVIEW

File: app/views/facilities/_members-table-row.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </td>
+  <td class="px-6 py-4 whitespace-nowrap text-center text-sm font-medium">
+    <% if policy(@facility).unassign_facility? %>
+      <%= form_with scope: :facility, url: unassign_facility_path, method: :put do |form| %>
+        <%= form.hidden_field "facility_id", value: @facility.id %>
+        <%= form.hidden_field "nurse_id", value:nurse.id %>
+        <%= button_tag "", {
+                      class: "bg-red-50 rounded-md p-1.5 text-red-500 hover:bg-red-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-red-50 focus:ring-red-600",
+                      onClick: "this.form.submit()"
+                      } do %>
```
Comment (shrihari689) [inline app/views/facilities/_members-table-row.html.erb:16]: I hope we can use `form.submit` which automatically submits on click. 🤔

File: app/views/facilities/show_users.html.erb [hunk 2]
```diff
@@ -4,90 +4,127 @@
...
+        <th
+          scope="col"
+          colspan="2"
+          class="px-6 py-3 text-center text-xs font-medium text-gray-500 uppercase tracking-wider"
+        >
+          Actions
+        </th>
+      </tr>
+    </thead>
+    <tbody x-max="2">
```
Comment (shrihari689) [inline app/views/facilities/show_users.html.erb:66]:
```suggestion
    <tbody>
```
I think this vue js thing which can from Tailwind Template 📍

File: app/views/facilities/show_users.html.erb [hunk 3]
```diff
@@ -4,90 +4,127 @@
...
```
Comment (shrihari689) [inline app/views/facilities/show_users.html.erb:113]: 📍

**Review by shrihari689 — COMMENTED** [review]
Looks good 🔥📌

File: app/views/facilities/_members-table-row.html.erb (hunk 1 above)
Comment (abhinandanarya06) [reply app/views/facilities/_members-table-row.html.erb:16]: See if `form.submit` supports enclosing svg like `button_tag`

Comment (shrihari689) [reply app/views/facilities/_members-table-row.html.erb:16]: Like this 👇👇

https://github.com/coronasafe/arike/blob/2f7f6334c3b4c64298ba4bf3c9c375df336b436a/app/views/devise/sessions/new.html.erb#L54

🔥

Comment (shrihari689) [reply app/views/facilities/_members-table-row.html.erb:16]: Like this 👇👇

https://github.com/coronasafe/arike/blob/main/app/views/devise/sessions/new.html.erb#L54:L59

🔥

Comment (abhinandanarya06) [reply app/views/facilities/_members-table-row.html.erb:16]: @ProCode2

**Comment by ProCode2** [comment]
Done! @abhinandanarya06 @shrihari689

**Review by abhinandanarya06 — APPROVED** [review]
Good job @ProCode2 . Now looks good to me. Approving the changes

**Pull request merged**